### DISCUSSION TITLE
increase in sound volume, 1 by 1

### DISCUSSION THREAD

**Discussion by DaLiCaM** [irzyxa/Volume2#336] [General]
Is it possible to configure the increase or decrease of the audio volume on the unit? Currently it does it 2 by 2

**Comment by DaLiCaM** [comment]
In previous versions I did not have this problem, it is worth mentioning that I have not changed hardware, I only reinstalled the operating system.
Operating System: Windows 10 Pro 64-bit (10.0, Build 19045)
System Manufacturer: Hewlett-Packard
Model: HP EliteDesk 800 G1 SFF
Memory: 16384MB RAM
Sound Devices: Realtek High Definition Audio (integrated)

**Reply by irzyxa** [reply]
You can do this by adding Hotkeys as shown below. Also change Volume changing speed to 100.
![image](https://github.com/user-attachments/assets/ef3cc031-4657-43de-99b2-c2dc403d00d2)
